src/NewTools-Inspector-Extensions/String.extension.st

```diff
@@ -27,7 +27,7 @@ String >> inspectionFullString: aBuilder [
 String >> inspectionString: aBuilder [
 
 	<inspectorPresentationOrder: -10 title: 'Preview'>
-	^ aBuilder newText
+	^ aBuilder newCode 
 		  text: (self printStringLimitedTo: 1000);
 		  yourself
```

Comment:
```suggestion
		  text: (self printStringLimitedTo: 1000);
		  beNotEditable;
		  yourself
```

We should not be able to edit this pane. This still allows to browse info but we cannot modify the test with this line